### PR TITLE
Added Zope pt, zpt and zcml

### PR DESCRIPTION
**Pull request by gotcha**

### CODE REVIEW

File: identify/extensions.py [hunk 1]
```diff
@@ -135,6 +135,7 @@
     'pp': {'text', 'puppet'},
     'properties': {'text', 'java-properties'},
     'proto': {'text', 'proto'},
+    'pt': {'text', 'zpt'},
```
Comment (asottile) [inline identify/extensions.py:138]: pt is likely too generic, a quick google search does not turn up zope but some other thing

Comment (gotcha) [reply identify/extensions.py:138]: Trying to understand what you mean: do you mean that `zpt` is too generic ? or that the `pt` extension is used by too many apps ?

If the latter, how does identification works when a file extension is used by different systems ?

Comment (asottile) [reply identify/extensions.py:138]: the latter

identification does not work in that case, we rejected extensions which are too uncommon or unofficial or ambiguous

Comment (gotcha) [reply identify/extensions.py]: I removed the `pt` registration.

As you might guess, my additions were made in the context of `pre-commit`.

Is it correct that to specify hooks for Zope page templates, usage of the `files` setting will then remain compulsory for `pt` files ? Or is there a better way ?

Comment (asottile) [reply identify/extensions.py]: yep, `files` will be necessary for that -- to be honest, `types` doesn't buy you much over `files` except that you avoid the regex (there's no shebangs involved which is where it really benefits)

**Review by asottile — APPROVED** [review]

**Pull request merged**